deeppavlov/models/preprocessors/torch_transformers_preprocessor.py

```diff
@@ -181,6 +174,50 @@ def __call__(self, texts_a: List[str], texts_b: Optional[List[str]] = None) -> U
         return input_features
 
 
+@register('torch_transformers_entity_ranker_preprocessor')
```

Comment:
remove @register: component didn't used from the configuration file (it's even doesn't added to the registry)

Reply:
added config for training where this component is used